### ISSUE TITLE
Iron Overlay type error on open in Chrome

### ISSUE THREAD

**Issue by valdrinkoshi** [PolymerLabs/polydev#31]
_From @fozziethebeat on October 29, 2015 18:28_

I've recently seen a problem in Chrome on both Ubuntu and Mac OSX where opening an iron-overlay throws a Javascript error:

```
Uncaught TypeError: Cannot read property 'toString' of null
```

The full stack trace looks like:    

```
obj.addEventListener    @   polydev/vendor/zone-microtask.js:973
Gestures.add    @   polymer.html:752 
Polymer.IronOverlayBehaviorImpl._toggleListener @   iron-overlay-behavior.html:268
(anonymous function)    @   iron-overlay-behavior.html:286
arguments.(anonymous function)  @   polydev/vendor/zone-microtask.js:406
(anonymous function)    @   polydev/vendor/zone-microtask.js:98
run @   polydev/vendor/zone-microtask.js:118
zoneBoundFn @   polydev/vendor/zone-microtask.js:91
```

I've seen this with both iron-dropdown and several custom elements I've used that mixin iron-overlay-behavior.

_Copied from original issue: PolymerElements/iron-overlay-behavior#47_

**Comment by valdrinkoshi** [comment]
_From @AndreasGalster on October 30, 2015 7:36_

Yes, I've had this issue for a while, thought it was an error related to my setup. You can reproduce the issue with this element for example: http://gabiaxel.github.io/paper-fab-transitions/components/paper-fab-transitions/demo/

Every element using iron-overlay-behavior is broken in at least Chrome Version 46.0.2490.71 m and above.

**Comment by valdrinkoshi** [comment]
_From @fozziethebeat on November 6, 2015 17:52_

I spoke with someone from the Polymer team and they commented that this is due to some interaction with the poly-dev toolkit.  So updating or removing that should address the problem.

**Comment by valdrinkoshi** [comment]
_From @btelles on January 6, 2016 18:41_

Bumping this as it's pretty annoying. Should we be reporting this in the Polymer Dev extension repository?

**Comment by valdrinkoshi** [comment]
cc @justinfagnani

**Comment by valdrinkoshi** [comment]
_From @justinfagnani on January 27, 2016 1:8_

I have the repro, which only happens with polydev. I'll look into this, but we can definitely move this to the polydev issues.

All of you who see this issue, you have polydev installed to a debugging-only Chrome profile, yes? :)

**Comment by Caffeinix** [comment]
@justinfagnani I'm still seeing this error.  The profile on which it's installed is indeed only used for debugging.

**Comment by rictic** [comment]
This should be fixed in v0.0.7 which was just merged in #35 and published. Please file a new issue if you see any more problems like this.
